### PR TITLE
Remove javascript scrollbar library, use hover appearing scrollbar instead

### PR DESCRIPTION
**Pull request by xPaw**
Removes the tse-scrollbar library. Makes the channel list and user list only have scrollbar visible when hovering over the container.

This library was used in the channel list only, it is sometimes quite laggy on mobile.

### CODE REVIEW

**Comment by astorije** [comment]
I think there were some issues discussed on the channel, but if we end up being able to get rid of that plugin, that's really cool! 🎉 

Will review when I get some free time.

**Comment by maxpoulin64** [comment]
I like that, works really well! Good job 👍 

Styling could be a bit better (at least color match the old tse one and a transition), but that can go with another PR.

File: client/css/style.css [hunk 1]
```diff
@@ -1557,62 +1549,22 @@ button,
...
-	position: relative;
-	overflow: hidden;
-}
-
-.tse-scrollable .tse-scroll-content {
-	overflow: hidden;
-	overflow-y: scroll;
+.scrollable {
+	overflow: auto;
+	visibility: hidden;
```
Comment (astorije) [inline client/css/style.css]: This has an unfortunate side effect, so I suggest removing fiddling with scrollbar visibility and letting the OS decide: scrollbars are set up to always show up on my OS, and this change effectively ignores this setup. Essentially, this is an accessibility issue (accessibility is not only about screen readers ;-) ).

**Comment by astorije** [comment]
One minor comment (disabling the display-on-hover) and then I'll be +1 on that PR :-)

**Comment by xPaw** [comment]
@astorije That's not exactly minor. Windows scrollbars are ugly in all browsers (except Chrome where we style them), and I'm sure users would want to get rid of them.

**Comment by astorije** [comment]
Right, I understand. However, I believe this should be a recipe in the CSS Mods page because, as it is in this PR, it contradicts OS settings in OSes that offer system-wide control of the scrollbars.

**Comment by xPaw** [comment]
@astorije I've removed the hover stuff.

**Comment by astorije** [comment]
Great @xPaw, 👍!
Glad we can get rid of an in-repo library 😄

**Pull request merged**